### ISSUE TITLE
Add test set

### ISSUE THREAD

**Issue by lzachmann** [lzachmann/DeepOSM#2]
As it stands, the `--percent-for-training-data` flag in create_training_data.py doesn't appear to do anything. As such, it's likely the case that the training data consists of _all_ of the data! Add support for this argument to permit a final accuracy assessment against an 'independent' (i.e., withheld) test set. 

Note that the validation_set argument to the `fit` method (called in single_layer_network.py) performs a data split over training data.
